migrations/20210612_initial.sql

```diff
@@ -54,7 +46,6 @@ CREATE TABLE groups (
   invited_by INTEGER REFERENCES contacts ON DELETE RESTRICT,
   external_group_id BLOB NOT NULL,
   local_group_ref TEXT NOT NULL UNIQUE, -- local group name without spaces
-  local_properties TEXT NOT NULL, -- local JSON group properties
```

Comment:
why remove this?

Reply:
we don't even have the use for profile properties yet, don't think we need these at all.